src/com/tony/server/response/RedirectResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.tony.server.response;
+
+import java.util.HashMap;
+
+public class RedirectResponse extends Response{
+    public RedirectResponse() {
+        setStatusLine("HTTP/1.1 302 Found\n");
+        HashMap<String, String> headers = new HashMap<>();
+        headers.put("Location", "http://localhost:5000/");
```

Comment:
In order to make this redirect response reusable, you should pass the new location in through the constructor.